scheduler/src/cook/components.clj

```diff
@@ -288,11 +277,12 @@
 (defn -main
   "Entry point for Cook. Initializes configuration settings,
   instruments the JVM, and starts up the scheduler and API."
-  [config & _]
+  [config-file-path & _]
+  (println "Cook" @util/version "( commit" @util/commit ")")
   (try
-    (let [settings (config/init-settings config)
-          _ (metrics-jvm/instrument-jvm)
-          server (scheduler-server settings)]
+    (mount/start-with-args (cook.config/read-config config-file-path))
```

Comment:
Could this be just the `config-file-path`? Then the necessary reading and initialization stays local to the `cook.config` namespace.

Reply:
I had it that way originally, but I thought it would be nice to be able to call `mount/start-with-args` with a map instead of a file (for example, when testing in the REPL).

Reply:
Okay